astropy/time/tests/test_update_leap_seconds.py

```diff
@@ -81,3 +83,20 @@ def test_auto_update_expired_file(self, tmpdir):
 
         with pytest.warns(iers.IERSStaleWarning):
             update_leap_seconds(['erfa', expired_file])
+
+    def test_init_thread_safety(self):
```

Comment:
The test looks good, but I'd prefer to make absolutely sure that the state after the test is the same as before. Maybe use pytest's `monkeypatch`?  Or a `try/finally` block?

Reply:
Done (with monkeypatch).